README.md

```diff
@@ -119,12 +119,12 @@ if !shouldDoTheThing {
 
 ---
 
-#####Use `where` to eliminate layers of nesting.
+#####Use a comma to eliminate layers of nesting.
```

Comment:
Do we think this section is even really that necessary any more?  I originally included it primarily because `where` was a new keyword and people weren't seeming to realize you could do this.

@lgauthier @Truked